core/src/databases_store/store.rs

```diff
@@ -182,33 +184,87 @@ impl DatabasesStore for PostgresDatabasesStore {
             }
         }
 
-        let stmt = c
-            .prepare(
-                "INSERT INTO tables_rows
-                   (table_id, row_id, created, content)
-                   SELECT * FROM UNNEST($1::text[], $2::text[], $3::bigint[], $4::text[])
-                   ON CONFLICT (table_id, row_id) DO UPDATE
-                   SET content = EXCLUDED.content",
-            )
+        let use_copy = true;
```

Comment:
I remove the flag but I updated so we only use the copy path if more than 1024 rows (just in case, that would limit the amount of tables that we would have to redo manually).